README.rst

```diff
@@ -1,8 +1,28 @@
-In Video Quiz XBlock |BS| |CA|
-==============================
+In Video Quiz XBlock
+====================
 
 This XBlock allows for edX components to be displayed to users inside of videos at specific time points.
```

Comment:
Any component or just problem/capa components? (or are all components implied to be problems?)

Reply:
Technically anything should work, though only used in practice with problem/capa.